services/libs/common_services/src/services/llm.service.ts

```diff
@@ -17,26 +16,26 @@ import {
   LlmMemberEnrichmentResult,
   LlmQueryType,
 } from '@crowd/types'
-
-import { LlmPromptHistoryRepository } from '../repos/llmPromptHistory.repo'
+import { insertPromptHistoryEntry } from '../../../data-access-layer/src/prompt-history'
+import { QueryExecutor } from '../../../data-access-layer/src/queryExecutor'
```

Comment:
It'll be cleaner and more maintainable if we import these with package imports
`import { insertPromptHistoryEntry, QueryExecutor } from '@crowd/data-access-layer'`
by installing the library in common_services